images/node/scripts/origin-node-run.sh

```diff
@@ -8,4 +8,4 @@ if [ "$#" -ne 0 ]; then
   opts=""
 fi
 
-exec /usr/bin/openshift start node "--config=${conf}" "${opts}" $@
+exec /usr/bin/openshift start node "--config=${conf}" ${opts} $@
```

Comment:
This is fine if we do not need to support values with whitespace.

/lgtm

However, if we do want to support values with whitespace (e.g., `OPTIONS=--foo='bar baz' --foo=bar\ baz --foo="bar baz"`), we need to do something a little fancier.  The following uses `eval` to perform shell word splitting on the value of `OPTIONS`:

```bash
eval "opts=( ${OPTIONS:---loglevel=2} )"
if [[ "$#" -ne 0 ]]; then
  opts=( "$@" )
fi

exec /usr/bin/openshift start node "--config=${conf}" "${opts[@]}"
```

Alternatively, the following is simpler in my opinion (but use your own judgment):

```bash
if [[ "$#" -eq 0 ]]; then
  eval "set -- ${OPTIONS:---loglevel=2}"
fi

exec /usr/bin/openshift start node "--config=${conf}" "$@"
```

Reply:
I am quite sure that OpenShfit/k8s will not support whitespace options like `OPTIONS=--foo='bar baz'`. However, it is not 0%, and also I don't want to overlook @Miciah's excellent idea, so updated the PR with his second option.